apps/SceneClustering/domset/nanoflann.hpp

```diff
@@ -0,0 +1,2039 @@
+/***********************************************************************
```

Comment:
however I would definitely place this file in Math lib, as this can easily be used in many other places

Reply:
resolved by bccdbe40c9e9a7eeb84811bb484d56eed82fd51d